commands/command.md

```diff
@@ -1,52 +1,87 @@
 Returns @array-reply of details about all Redis commands.
 
-Cluster clients must be aware of key positions in commands so commands can go to matching instances,
-but Redis commands vary between accepting one key,
-multiple keys, or even multiple keys separated by other data.
+Cluster clients must be aware of key positions in commands so commands can go to matching instances, but Redis commands vary between accepting one key, multiple keys, or even multiple keys separated by other data.
+
+Starting from Redis 7.0 a more flexible mechanism was introduced in order to help clients finding the key positions.
+For more information please check the [key-specs page][tr].
+[tr]: /topics/key-specs
 
 You can use `COMMAND` to cache a mapping between commands and key positions for
 each command to enable exact routing of commands to cluster instances.
 
-## Nested Result Array
-Each top-level result contains six nested results.  Each nested result is:
-
-  - command name
-  - command arity specification
-  - nested @array-reply of command flags
-  - position of first key in argument list
-  - position of last key in argument list
-  - step count for locating repeating keys
-
-### Command Name
-
-Command name is the command returned as a lowercase string.
-
-### Command Arity
-
-<table style="width:50%">
-<tr><td>
-<pre>
-<code>1) 1) "get"
+`COMMAND` has several subcommands:
+ - `COMMAND INFO`
+ - `COMMAND COUNT`
+ - `COMMAND GETKEYS`
+ - `COMMAND LIST`
+
+`COMMAND`'s reply is an @array-reply, where each element is an @array-reply by itself, containing the following elements:
+ - command name
+ - command arity specification
+ - nested @array-reply of command flags
+ - position of first key in argument list
+ - position of last key in argument list
+ - step count for locating repeating keys
+ - nested @array-reply of ACL categories
+ - nested @map-reply of additional information
```

Comment:
The current website doesn't support RESP3 return types, nor does it feature the RESP3 spec :/
```suggestion
 - nested @array-reply of additional information as a map
```